tasks/sendBTC.ts

```diff
@@ -0,0 +1,148 @@
+import confirm from "@inquirer/confirm";
+import * as bitcoin from "bitcoinjs-lib";
+import * as dotenv from "dotenv";
+import ECPairFactory from "ecpair";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import * as ecc from "tiny-secp256k1";
+
+dotenv.config();
+
+const TESTNET = bitcoin.networks.testnet;
+
+const API = "https://blockstream.info/testnet/api";
+const ENDPOINT = "https://api.blockcypher.com/v1/btc/test3/txs";
+
+type UTXO = {
+  txid: string;
+  value: number;
+  vout: number;
+};
+
+const decodeTransaction = async (tx: any) => {
+  const endpoint = `${ENDPOINT}/decode`;
+
+  const p1 = await fetch(endpoint, {
+    body: JSON.stringify({ tx }),
+    method: "POST",
+  });
+  return await p1.json();
+};
+
+const fetchUtxos = async (address: string): Promise<UTXO[]> => {
+  const response = await fetch(`${API}/address/${address}/utxo`);
+  return response.json();
+};
+
+const makeTransaction = async (
+  to: string,
+  key: any,
+  amount: any,
+  utxos: any,
+  address: string,
+  memo: string = ""
```

Comment:
later, it would be cool if we have a tool to help you construct the memo for btc (cli tool like this, or a helper function)!

Reply:
https://github.com/zeta-chain/toolkit/issues/16